src/dev/flang/fuir/analysis/dfa/DFA.java

```diff
@@ -2068,6 +2058,25 @@ static void setArrayElementsToAnything(Call cl, int argnum, String intrinsicName
     put("f64.type.min_positive"          , cl -> NumericValue.create(cl._dfa, cl._dfa._fuir.clazzResultClazz(cl._cc)) );
     put("f64.type.max"                   , cl -> NumericValue.create(cl._dfa, cl._dfa._fuir.clazzResultClazz(cl._cc)) );
     put("f64.type.epsilon"               , cl -> NumericValue.create(cl._dfa, cl._dfa._fuir.clazzResultClazz(cl._cc)) );
+    put("effect.type.get"                , cl ->
+    {
+      var ecl = cl._dfa._fuir.clazzResultClazz(cl._cc);
+      var result = cl.getEffectCheck(ecl);
+      if (result == null && cl._dfa._reportResults)
+        {
+          DfaErrors.usedEffectNotInstalled(cl._dfa._fuir.sitePos(cl._site),
+                                           cl._dfa._fuir.clazzAsString(ecl),
+                                           cl);
+          cl._dfa._missingEffects.put(ecl, ecl);
+        }
+      return result;
+    });
```

Comment:
you should either usee `Call.getEffectForce` here or remove `Call.getEffectForce` which is otherwise unused.